index.html

```diff
@@ -22,12 +22,16 @@
           <img src="https://bulma.io/images/made-with-bulma.png" alt="Made with Bulma" width="128" height="24">
         </a>
         <br>
-        <small>Copyright © 2017 by <a href="https://github.com/mazipan/" target="_blank">Irfan Maulana</a></small>
+        <small>Copyright © <span id="yearNow"></span> by <a href="https://github.com/mazipan/" target="_blank">Irfan Maulana</a></small>
```

Comment:
can we make 2017-yearNow.
Because I need to show the year of created.

Reply:
Right. On it.